docs/concepts/policy/pod-security-policy.md

```diff
@@ -207,37 +207,58 @@ podsecuritypolicy "permissive" deleted
 
 ## Enabling Pod Security Policies
 
-In order to use Pod Security Policies in your cluster you must ensure the
-following
+In order to use Pod Security Policies in your cluster:
 
-1.  You have enabled the API type `extensions/v1beta1/podsecuritypolicy` (only for versions prior 1.6)
-1.  [You have enabled the admission control plug-in `PodSecurityPolicy`](/docs/admin/admission-controllers/#how-do-i-turn-on-an-admission-control-plug-in)
-1.  You have defined your policies
+1. Enable the API type `extensions/v1beta1/podsecuritypolicy` (only for versions prior 1.6)
+1. [Enable the admission control plug-in `PodSecurityPolicy`](/docs/admin/admission-controllers/#how-do-i-turn-on-an-admission-control-plug-in)
+1. Create PodSecurityPolicy objects
+1. Authorize users, groups, or service accounts to perform the `use` verb on desired policies
 
 ## Working With RBAC
 
 In Kubernetes 1.5 and newer, you can use PodSecurityPolicy to control access to
 privileged containers based on user role and groups. Access to different
 PodSecurityPolicy objects can be controlled via authorization.
 
-Note that [Controller Manager](/docs/admin/kube-controller-manager/) must be run
-against [the secured API port](/docs/admin/accessing-the-api/), and must not
-have superuser permissions. Otherwise requests would bypass authentication and
-authorization modules, all PodSecurityPolicy objects would be allowed,
-and user will be able to create privileged containers.
+Before beginning to grant access to specific policies, ensure users and service accounts have not been given global access
+by running the following commands as a privileged user.
```

Comment:
I'm afraid that some users would misunderstand it and will run this command under `root` user instead. Could you point them that "privileged user" means "a user with `cluster-admin` role"?